lib/krikri/search_index.rb

```diff
@@ -33,11 +33,10 @@ def bulk_add(_)
     # Shim that determines, for a particular type of index, which strategy
     # to use, adding a single document, or adding them in bulk.  Intended
     # to be overridden as necessary.
+    #
     # @see #add
     # @see #bulk_add
     def update_from_activity(activity)
-      fail "#{activity} is not an Activity" \
```

Comment:
Note that I've simply removed this, relying on duck typing instead.  Anything with `#entities` will work fine, here.